base/src/main/java/com/smartdevicelink/proxy/rpc/OnDriverDistraction.java

```diff
@@ -107,5 +109,37 @@ public DriverDistractionState getState() {
      */    
     public void setState( @NonNull DriverDistractionState state ) {
         setParameters(KEY_STATE, state);
-    }  
+    }
+
+    /**
+     * <p>Called to set dismissible state of Lockscreen</p>
+     * @param isDismissible the Lockscreen'S dismissibility
+     */
+    public void setLockscreenDismissible(boolean isDismissible) {
+        setParameters(KEY_LOCKSCREEN_DISMISSIBLE, isDismissible);
+    }
+
+    /**
+     * <p>Called to get the dismissible state of Lockscreen</p>
+     * @return true if the Lockscreen is dismissible, false otherwise
+     */
+    public Boolean getLockscreenDismissibility() {

```

Comment:
The setter and getter method naming should match for the same key

Reply:
Done